R/save_multiple_laterals.R

```diff
@@ -0,0 +1,68 @@
+save_lateral_yards <- function(s) {
+  future::plan("multisession")
+
+  games <- nflreadr::load_schedules(s) |>
+    dplyr::filter(!is.na(result))
+
+  if(length(games$game_id) == 0){
+    cli::cli_alert_info("No games yet, nothing to do")
+    return(invisible(NULL))
+  }
+
+  cli::cli_alert_info("Load {.val {length(games$game_id)}} game{?s} of {.val {s}}")
+
+  load <- furrr::future_map_dfr(games$game_id, check_lateral_yards)
+
+  cli::cli_alert_info("Process raw data...")
+
+  all <-
+    dplyr::bind_rows(
+      readr::read_csv("lateral_yards/multiple_lateral_yards.csv", show_col_types = FALSE),
+      load |>
+        nflfastR::decode_player_ids() |>
+        dplyr::group_by(game_id, play_id) |>
+        dplyr::filter(dplyr::n() > 1) |>
+        dplyr::ungroup() |>
+        suppressMessages()
+    ) |>
+    dplyr::distinct() |>
+    dplyr::arrange(game_id, play_id)
+
```

Comment:
now that I think about it, it would be useful if everything happened in this one action rather than having two separate actions. 

pseudocode:
```r
old<- readr::read_csv("lateral_yards/multiple_lateral_yards.csv", show_col_types = FALSE)

if( nrow(old) == nrow(all)) {
  nflversedata::nflverse_save()
}
# write csv
```

Reply:
and then get rid of update_multiple_laterals and the other GHA. 

This prevents needing to install shit in a new runner etc